apps/charterafrica/src/payload/fields/ecosystem/airtableBaseSelect.js

```diff
@@ -8,6 +8,15 @@ import fetchJson from "../../../utils/fetchJson";
 
 const url = `${process.env.PAYLOAD_PUBLIC_APP_URL}/api/v1/resources/ecosystem/schema?source=airtable&url=/meta/bases`;
 
+export function getUrl(baseId) {
```

Comment:
1. Why do we have `url` and getUrl? Seems like `getUrl` should probably be `getTablesUrl` (or similar)
2. `getTablesUrl` (or similar) can reuse `url` and just append `/${baseId}/tables`, no?